src/main/java/uk/gov/companieshouse/insolvency/data/converter/CaseDateTypeEnumReadingConverter.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.companieshouse.insolvency.data.converter;
+
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.data.convert.ReadingConverter;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.insolvency.CaseDates;
+import uk.gov.companieshouse.api.insolvency.ModelCase;
```

Comment:
We should have unit tests for these classes as well as code coverage is already below 50% unfortunately